### PR TITLE
docs(buttons): add keyboard navigation

### LINKED ISSUE #3987: docs(buttons): add keyboard navigation
Need to add key navigation feature to each demo with group of buttons. This buttons are indicated by a tag `label`. Needs to add to them:
- tabindex
- role

effort1: easy(hours)

### PR DESCRIPTION
**Pull request by tarusin**
After adding tabindex and role attributes in buttons group which marked by `label` tag they have focus by `tab` switching. And screen readers see them like buttons.

Closes #3987

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+buttons/demos/checkbox/checkbox.html [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 <div class="btn-group">
   <label class="btn btn-primary" [(ngModel)]="checkModel.left"
-         btnCheckbox>Left</label>
-  <label class="btn btn-primary" [(ngModel)]="checkModel.middle" btnCheckbox>Middle</label>
-  <label class="btn btn-primary" [(ngModel)]="checkModel.right" btnCheckbox>Right</label>
+         btnCheckbox tabindex="0" role="button">Left</label>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+buttons/demos/checkbox/checkbox.html]: Nothing happens when I focus on button and press `Enter` or `Space`. This is about all buttons with role and tabindex, not just this one

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Now not clear, how to test this PR (how can these changes improve accessibility). Please, add info what keys can be used for navigation and how. Or, it will be the clearest way, add steps how to test in issue, or in PR's description

**Comment by YevheniiaMazur** [comment]
@tarusin , do I need special screenreader for testing this PR, or any screenreader can see buttons now?

**Comment by tarusin** [comment]
@YevheniiaMazur, you can use browser extantion chromeVox to see difference.

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**